apps/admin/backend/src/store.ts

```diff
@@ -790,6 +842,7 @@ export class Store {
         `,
         electionId
       );
+      this.deleteEmptyScannerBatches(electionId);
```

Comment:
More out of curiosity: is there a case where we couldn't just wipe all the batches for `electionId` after deleting all CVRs? (i.e. when is a batch non-empty for an election after deleting the CVRs?)

Reply:
In this case you could do that. When I wrote the helper I was also thinking of the case of removing only a single CVR file which we plan to support: https://votingworks.slack.com/archives/CEL6D3GAD/p1682555456417949.

On that topic, it's a simple change so I'm just going to leave it undone until we build the frontend for it.

Reply:
Ah good point, I remember browsing that thread - makes sense to keep as is